pkg/processor/deployment/deployment.go

```diff
@@ -129,7 +130,8 @@ func (d deployment) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstr
 		return true, nil, err
 	}
 
-	spec = strings.ReplaceAll(spec, "'", "")
+	r := regexp.MustCompile(`'({{((.*|.*\n.*))}}.*)'`)
+	spec = r.ReplaceAllString(spec, "${1}")
```

Comment:
it looks like exacltly what we need but for me it is always hard read regexp. Can you please move this `'` replacement code to a separate function and add table unit test covering edge cases?